pkg/kubectl/cmd/util/cached_discovery.go

```diff
@@ -25,8 +25,8 @@ import (
 	"time"
 
 	"github.com/emicklei/go-restful-swagger12"
-	"github.com/go-openapi/spec"
 	"github.com/golang/glog"
+	openapi_v2 "github.com/googleapis/gnostic/OpenAPIv2"
```

Comment:
wondering if we have any convention for package import names ? using "_" is a standard thing (I see metav1 below) ?